plugin.rb

```diff
@@ -173,16 +153,11 @@ def alert_low_votes?
       end
 
       def votes
-        votes = self.custom_fields[DiscourseVoting::VOTES] || []
-        # "" can be in there sometimes, it gets turned into a 0
-        votes = votes.reject { |v| v == 0 }.uniq
-        votes
+        self.topic_votes.where(archive: false).pluck(:topic_id)
```

Comment:
Slight nitpick but the method name and the return value is confusing to me. I would have expected the method to return AR records of `DiscourseVoting::Vote` instead of the topic ids.